content/reference/cpp/string-view.md

```diff
@@ -0,0 +1,186 @@
++++
+title = "String View APIs"
+weight = 510
+linkTitle = "Generated Code Guide"
```

Comment:
I just realized this may be a copy-paste bug from the docs this was branched from...

Reply:
You're right. I'll patch in an update.

Reply:
This is now updated.